src/app/clients/form/submitFormHelpers.ts

```diff
@@ -60,13 +58,19 @@ const formatClientRecord = (
         extra_information: extraInformationWithNappy,
         signposting_call_required: fields.signpostingCall,
         flagged_for_attention: fields.attentionFlag,
+        last_updated: fields.lastUpdated,
     };
 };
 
-export const submitAddClientForm = async (
-    fields: ClientFields,
-    router: AppRouterInstance
-): Promise<void> => {
+type addClientErrors = "failedToInsertClientAndFamily";
+type addClientReturnType =
```

Comment:
For convention, we'll stick to `...Result` :)
```suggestion
type addClientResult =
```